travis/test_server.py

```diff
@@ -350,12 +351,12 @@ def main(argv=None):
 
     if odoo_unittest:
         to_test_list = tested_addons_list
-        cmd_odoo_install = [
-            "%s/%s" % (server_path, script_name),
-            "-d", database,
-            "--stop-after-init",
-            "--log-level=warn",
-        ] + install_options + ["--init", None] + server_options
+        cmd_odoo_install = ["%s/openerp-server" % server_path,
+                            "-d", database,
+                            "--stop-after-init",
+                            "--log-level=warn",
+                            ] + install_options + ["--init", None] +\
+                            server_options
```

Comment:
Actually I haven't introduced them. This is what I introduced besides of reverting the previous commit: https://github.com/OCA/maintainer-quality-tools/pull/409/commits/9696b3aba595c810f7a26fa08bfd6a9b57655d65
Remember that @StefanRijnhart has agreed to revert it. At least that's what his [comment](https://github.com/OCA/maintainer-quality-tools/issues/398#issuecomment-267426594) suggests. I see now, how complicated distributed communication and coordination is 😄. I'm realizing this, while trying to keep out of the line of fire somehow...